g4f/Provider/bing/create_images.py

```diff
@@ -194,48 +178,47 @@ def get_cookies_from_browser(proxy: str = None) -> Dict[str, str]:
 
 class CreateImagesBing:
     """A class for creating images using Bing."""
-
-    _cookies: Dict[str, str] = {}
     
-    @classmethod
-    def create_completion(cls, prompt: str, cookies: Dict[str, str] = None, proxy: str = None) -> Generator[str, None, None]:
+    def __init__(self, cookies: dict[str, str] = {}, proxy: str = None) -> None:
+        self.cookies = cookies
+        self.proxy = proxy
+    
+    def create_completion(self, prompt: str) -> Generator[ImageResponse, None, None]:
```

Comment:
Consider adding type hints to the `prompt` parameter of the `create_completion` method.